### PR TITLE
Typo + prizes

### PR DESCRIPTION
**Pull request by Mussuranae**
Correct some typo on movie and distribution right collection.
Also move every prizes into the new customPrizes field because of the introduction of the new static about festival prizes.

### CODE REVIEW

File: libs/movie/src/lib/movie/+state/movie.firestore.ts [hunk 1]
```diff
@@ -51,6 +51,7 @@ interface MovieRaw<D> {
   contentType: ContentType; //! required
   crew?: Crew[],
   customGenres?: string[],
+  customPrizes: Prize[],
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/+state/movie.firestore.ts:54]: ` customPrizes: Prize[]`, was better after  `prizes?: Prize[],`

File: libs/import/src/lib/components/view-extracted-elements/movies/view-extracted-movies.component.ts [hunk 2]
```diff
@@ -501,11 +501,7 @@ export class ViewExtractedMoviesComponent implements OnInit {
                 }
 
               }
-              // TODO issue #3091
-              // if (prizeParts.length >= 5) {
-              //   prize.logo = await this.imageUploader.upload(prizeParts[4].trim());
-              // }
```
Comment (phpgeekfr) [inline libs/import/src/lib/components/view-extracted-elements/movies/view-extracted-movies.component.ts:507]: We want to keep this comment

File: libs/movie/src/lib/movie/+state/movie.model.ts [hunk 3]
```diff
@@ -53,6 +53,7 @@ export function createMovie(params: Partial<Movie> = {}): Movie {
     documents: createMovieLegalDocuments(params.documents),
     // Mandatory fields
     contentType: 'feature_film',
+    customPrizes: [],
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/+state/movie.model.ts:56]: customPrizes: Prize[], was better after prizes?: Prize[],

File: libs/import/src/lib/components/view-extracted-elements/movies/view-extracted-movies.component.ts [hunk 4]
```diff
@@ -482,7 +482,7 @@ export class ViewExtractedMoviesComponent implements OnInit {
 
         // PRIZES (Prizes)
         if (spreadSheetRow[SpreadSheetMovie.festivalPrizes]) {
-          movie.prizes = [];
+          movie.customPrizes = [];
```
Comment (phpgeekfr) [inline libs/import/src/lib/components/view-extracted-elements/movies/view-extracted-movies.component.ts:485]: update also `if (movie.prizes.length === 0) {`  in `validateMovie`

Comment (phpgeekfr) [reply libs/import/src/lib/components/view-extracted-elements/movies/view-extracted-movies.component.ts:485]: and maybe rename spreadSheetRow[SpreadSheetMovie.festivalPrizes to better understanding

File: libs/movie/src/lib/movie/form/movie.form.ts [hunk 5]
```diff
@@ -94,6 +94,7 @@ function createMovieControls(movie: Partial<Movie>) {
     contentType: new FormControl(entity.contentType, [Validators.required]),
     crew: FormList.factory(entity.crew, el => new CreditForm(el)),
     customGenres: FormList.factory(entity.customGenres, el => new FormControl(el)),
+    customPrizes: FormList.factory(entity.customPrizes, el => new MoviePrizeForm(el)),
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/movie.form.ts:97]: you prefer grouping Custom fields ?

File: libs/movie/src/lib/movie/form/summary/review/review.component.html [hunk 6]
```diff
@@ -40,14 +40,18 @@ <h6>
...
       <mat-icon svgIcon="pencil" color="primary"></mat-icon>
     </a>
   </h6>
   <ul>
-    <ng-container *ngIf="movie.customPrize.controls.length">
-      <li *ngFor="let customPrize of movie.customPrize.controls">
+    <ng-container *ngIf="!movie.customPrizes.controls.length else displayCustomFestival">
+      <a [routerLink]="link" fragment="custom-festival" class="missing-control">
+        Missing
+      </a>    </ng-container>
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/summary/review/review.component.html:52]: bad indent

File: apps/backend-ops/src/firestoreMigrations/0036.ts [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+  movies.docs.map(movieDoc => {
+    const movie = movieDoc.data();
+
+    delete movie.releaseYear;
+    delete movie.totalRunTime;
+
+    const newMovieData = {
+      ...movie,
+      customPrizes: movie.prizes,
+      prizes: [],
```
Comment (phpgeekfr) [inline apps/backend-ops/src/firestoreMigrations/0036.ts:26]: Every old movies.prizes goes into customPrizes ?

Comment (phpgeekfr) [reply apps/backend-ops/src/firestoreMigrations/0036.ts:26]: There is no existing `customPrize` that should be removed?

**Review by phpgeekfr — APPROVED** [review]

File: libs/import/src/lib/components/view-extracted-elements/movies/view-extracted-movies.component.ts (hunk 2 above)
Comment (Mussuranae) [reply libs/import/src/lib/components/view-extracted-elements/movies/view-extracted-movies.component.ts:507]: But at least, the logo for the prizes will be manage by C8 and not by the user (that's also why we introduce the new static model), I will add it again, but normally user won't be able to upload logo for festival. We can ask Vincent to be sure about that.

Comment (phpgeekfr) [reply libs/import/src/lib/components/view-extracted-elements/movies/view-extracted-movies.component.ts:507]: OK thanks. You can remove it then

File: apps/backend-ops/src/firestoreMigrations/0036.ts (hunk 7 above)
Comment (Mussuranae) [reply apps/backend-ops/src/firestoreMigrations/0036.ts:26]: No, it's a new field, and as we will use now the prizes with a static list, I thought that all prizes we have would be better t be in the customPrizes.

File: libs/movie/src/lib/movie/form/movie.form.ts (hunk 5 above)
Comment (Mussuranae) [reply libs/movie/src/lib/movie/form/movie.form.ts:97]: I'm just inflexible and I ordered everything alphabetically.... I did an effort for you x)

**Pull request merged**